client/main.lua

```diff
@@ -37,7 +37,8 @@ local function raycastWeapon()
     local num = math.abs(math.cos(direction.x))
     direction = vec3((-math.sin(direction.y) * num), (math.cos(direction.y) * num), math.sin(direction.x))
     local destination = vec3(offset.x + direction.x * 30, offset.y + direction.y * 30, offset.z + direction.z * 30)
-    local rayHandle, result, hit, _, _, entityHit = StartShapeTestLosProbe(offset, destination, -1, playerPed, 0)
+	local hit, entityHit, result
+    local rayHandle, _, _, _, _, _ = StartShapeTestLosProbe(offset, destination, -1, playerPed, 0)
```

Comment:
```suggestion
    local rayHandle = StartShapeTestLosProbe(offset, destination, -1, playerPed, 0)
```
`StartShapeTestLosProbe` doesn't return anything else so no need for all those underscores, even if it did there was no need unless a variable was used in between

Reply:
Okay. I wasn't aware of that, will remove the underscores